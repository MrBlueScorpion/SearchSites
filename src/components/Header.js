/**
 * Created by Lixing on 14/1/17.
 */
import React from 'react';
import Logo from '../assets/logo.png';

const Header = () => (
  <div style={style.container}>
    <nav style={style.nav}>
      <img src={Logo} alt="Adslot" />
    </nav>
  </div>
);

const style = {
  container: {
    display: 'flex',
    height: 80,
    borderBottomWidth: 1,
    borderBottomColor: '#d8d8d9',
    borderBottomStyle: 'solid',
    justifyContent: 'center',
    alignItems: 'center'

  },
  nav: {
    width: 1170
  }

};

export default Header;
