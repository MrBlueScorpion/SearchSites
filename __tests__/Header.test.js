/**
 * Created by Lixing on 15/1/17.
 */
import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Header from '../src/components/Header';

describe('<Header />', () => {
  it('should render <Header /> component', () => {
    const wrapper = shallow(<Header />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});