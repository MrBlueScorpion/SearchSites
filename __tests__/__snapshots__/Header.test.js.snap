exports[`<Header /> should render <Header /> component 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "borderBottomColor": "#d8d8d9",
      "borderBottomStyle": "solid",
      "borderBottomWidth": 1,
      "display": "flex",
      "height": 80,
      "justifyContent": "center",
    }
  }>
  <nav
    style={
      Object {
        "width": 1170,
      }
    }>
    <img
      alt="Adslot"
      src="logo.png" />
  </nav>
</div>
`;
